library/src/main/java/com/getstream/sdk/chat/viewmodel/ChannelViewModel.java

```diff
@@ -59,10 +62,12 @@
     private Map<String, Event> typingState;
 
     private int channelSubscriptionId = 0;
+    private int threadParentPosition = 0;
```

Comment:
what is this for?

Reply:
This is for scroll up origin position when back from Thread.